packages/web-components/fast-foundation/src/accordion/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    attr,
+    FASTElement,
+    nullableNumberConverter,
+    observable,
+} from "@microsoft/fast-element";
+import { StartEnd } from "../../patterns/start-end";
+import { applyMixins } from "../../utilities/apply-mixins";
+
+export class AccordionItem extends FASTElement {
+    @attr({
+        attribute: "heading-level",
+        mode: "fromView",
+        converter: nullableNumberConverter,
+    })
+    public headinglevel: 1 | 2 | 3 | 4 | 5 | 6 = 2;
+
+    @attr({ mode: "boolean" })
+    public expanded: boolean = false;
+
+    @attr
+    public id: string;
+
+    @observable
```

Comment:
No need to put an `@observable` decorator on a ref property. Basically, you only need `@observable` if the template needs to update when your code changes the property. In the case of ref, it's the template that's setting you (other way around).